SETUP.md

```diff
@@ -111,4 +111,4 @@ Docker.Core.DockerExceptionFilesharing has been cancelled
 operable program or batch file.
 
 ```   
-* Fix: Run ```npm ci```  
```

Comment:
I'm not sure why it shows this last line being added; it's in the original document here https://github.com/rtcharity/eahub.org/blob/default/SETUP.md